plugins/gnostic-vocabulary/sample-petstore.out

```diff
@@ -0,0 +1,86 @@
+
+
+/usr/local/google/home/nicolegizzo/Desktop/fields/gnostic/examples/v2.0/yaml/vocabulary.json -------------------- 
+{
+  "schemas": [
+    {
+      "word": "Error",
+      "count": 1
+    },
+    {
+      "word": "Pet",
+      "count": 1
+    },
+    {
+      "word": "Pets",
+      "count": 1
+    }
+  ],
+  "properties": [
+    {
+      "word": "id",
+      "count": 1
+    },
+    {
+      "word": "name",
+      "count": 1
+    },
+    {
+      "word": "tag",
+      "count": 1
+    },
+    {
+      "word": "code",
+      "count": 1
+    },
+    {
+      "word": "message",
+      "count": 1
+    }
+  ],
+  "operations": [
+    {
+      "word": "listPets",
+      "count": 1
+    },
+    {
+      "word": "createPets",
+      "count": 1
```

Comment:
I thought these words would be sorted alphabetically. Maybe I'm not seeing your latest change?